src/ServiceManagement/Common/Commands.ServiceManagement.Common/RequiredResourceLookup.cs

```diff
@@ -91,6 +91,11 @@ internal static IList<string> RequiredProvidersForResourceManager<T>()
                 return new[] { StorageProviderNamespace };
             }
 
+            if (typeof(T).FullName.Equals("Microsoft.Azure.Management.Network.NetworkManagementClient"))
```

Comment:
what is this for?

Reply:
For creating and accessing the Network Management client.

Reply:
This is in the ServiceManagement path/namespace. Is it used in the ResourceManager path/namespace?

Reply:
Not exactly sure what you mean. Let me test it after removing it, probably I won't even need it.